### PR TITLE
Transcript and Certificate Merge Process

### PR DESCRIPTION
**Pull request by sreepadkarekkad**

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -165,6 +165,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.pdfbox</groupId>
+            <artifactId>pdfbox</artifactId>
+            <version>2.0.1</version>
```
Comment (kamal-mohammed) [inline api/pom.xml:171]: We need the version 2.0.24 or 2.0.25 as the previous versions have some High Risk vulnerabilities reported.

Comment (sreepadkarekkad) [reply api/pom.xml:171]: Sure, i will change that right now and push it

File: api/src/main/java/ca/bc/gov/educ/api/distribution/process/MergeProcess.java [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+				setExtraDataForPackingSlip(packSlipReq,"YED4",obj.getTotal(),scdList.size(),1,"Transcript",transcriptPrintRequest.getBatchId());
+				try {
+					locations.add(reportService.getPackingSlip(packSlipReq,processorData.getAccessToken(),exception).getInputStream());
+					logger.info("Packing Slip Added");
+					for (StudentCredentialDistribution scd : scdList) {
+						InputStreamResource transcriptPdf = webClient.get().uri(String.format(educDistributionApiConstants.getTranscript(),scd.getStudentID(),scd.getCredentialTypeCode(),"COMPL")).headers(h -> h.setBearerAuth(processorData.getAccessToken())).retrieve().bodyToMono(InputStreamResource.class).block();
+						locations.add(transcriptPdf.getInputStream());
+					}
+					PDFMergerUtility objs = new PDFMergerUtility();
+					Path path = Paths.get("C:/Users/s.karekkattumanasree/Downloads/"+mincode+"/");
```
Comment (kamal-mohammed) [inline api/src/main/java/ca/bc/gov/educ/api/distribution/process/MergeProcess.java:91]: This path can be /tmp/mincode/ for now when it runs on Openshift. We'll work on setting up the volume for that.

**Review by kamal-mohammed — APPROVED** [review]

File: api/src/main/java/ca/bc/gov/educ/api/distribution/process/MergeProcess.java (hunk 2 above)
Comment (sreepadkarekkad) [reply api/src/main/java/ca/bc/gov/educ/api/distribution/process/MergeProcess.java:91]: will change it :)

**Pull request merged**